### ISSUE TITLE
cannot get verified with pocket

### ISSUE THREAD

**Issue by januario-hh** [Adam42/pull-tabs#34]
I cannot configure and thus not use at all this extension, because I cannot use the link "you are not authorized, please sign in" (under Services > Pocket). Clicking on it does nothing.

**Comment by Adam42** [comment]
Could you let me know what browser you're using and what version of the extension?

**Comment by januario-hh** [comment]
the latest Firefox (57.0, 64 bit) and pulltab 0.11.0. I have manually typed the underlying link which can be viewed in the left downward screen corner into the address field and could then access/authorize pocket. Then I could eventually proceed to save the tabs. My acknowledgement: This seems to be really helpful! Thank you!
/J.

**Comment by Adam42** [comment]
Glad you found a workaround! 

When I started the Firefox version of this extension Mozilla did not yet have a version of launchWebAuthFlow https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/identity/launchWebAuthFlow so I had to add custom functionality. Now that launchWebAuthFlow is available in Firefox I'll be updating the extension to use it which should make authenticating to Pocket and other services much less flaky.

Will update this issue when that change makes it into a new release of the extension.
